.github/workflows/fidelity-tests.yml

```diff
@@ -1,14 +1,14 @@
 name: Fidelity tests
 
-on: [pull_request]
+on: [pull_request, push]
```

Comment:
Do you want fidelity test to run jus on pull request is made , or run every time a new push is made?

Reply:
I think just pull request is fine, unless you have a strong opinion.